torch/nn/utils/rnn.py

```diff
@@ -254,7 +261,7 @@ def pack_padded_sequence(input, lengths, batch_first=False, enforce_sorted=True)
                       'sequence lengths. The tracer cannot track the data flow of Python '
                       'values, and it will treat them as constants, likely rendering '
                       'the trace incorrect for any other combination of lengths.',
-                      category=torch.jit.TracerWarning, stacklevel=2)
+                      stacklevel=2)
```

Comment:
Can't we make JIT ignore the category? It's not a good idea to remove it

Reply:
category is hard to ignore atm because we don't have any way of typing `torch.jit.TracerWarning`

Reply:
yeah I tried that and it's kind of hard now, plus other callsites does not have this category either way.. not sure if someone deleted them before.